front/src/common/SpeedLimitByTagSelector/useStoreDataForSpeedLimitByTagSelector.ts

```diff
@@ -43,12 +41,14 @@ export const useStoreDataForSpeedLimitByTagSelector = () => {
     }
   }, [error]);
 
-  let speedLimitsByTags = compact(uniq(concat(speedLimitTags, speedLimitsTagsByInfraId)));
+  const speedLimitsByTags = compact(uniq(concat(speedLimitTags, speedLimitsTagsByInfraId)));
   const speedLimitsByTagsOrdered = useMemo(() => speedLimitsByTags.sort(), [speedLimitsByTags]);
```

Comment:
Random drive-by comment: this `useMemo` never memorizes anything because its input changes at each render.

(But that's unrelated to this PR.)